src/lua-function-list.json

```diff
@@ -424,5 +445,6 @@
     "uninstallPackage": "uninstallPackage(name)",
     "unzipAsync": "unzipAsync(path, location)",
     "updateMap": "updateMap()",
+    "usingMudletsDiscordID": "usingMudletsDiscordID(state)",
```

Comment:
```suggestion
    "usingMudletsDiscordID": "usingMudletsDiscordID()",
```

Reply:
This has to be corrected in the wiki, not here!

Reply:
I've done that